services/merak-network/test/create_test/network_create_try.go

```diff
@@ -0,0 +1,192 @@
+/*
+MIT License
+Copyright(c) 2022 Futurewei Cloud
+    Permission is hereby granted,
+    free of charge, to any person obtaining a copy of this software and associated documentation files(the "Software"), to deal in the Software without restriction,
+    including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and / or sell copies of the Software, and to permit persons
+    to whom the Software is furnished to do so, subject to the following conditions:
+    The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+    WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"time"
+
+	pb "github.com/futurewei-cloud/merak/api/proto/v1/merak"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+const (
+	defaultName = "world"
+)
+
+var (
+	addr = flag.String("addr", "localhost:40053", "the address to connect to")
+	name = flag.String("name", defaultName, "Name to greet")
+)
+
+func main() {
+	testInternalSecurityGroupRulelnfo := pb.InternalSecurityGroupRulelnfo{
+		OperationType: pb.OperationType_CREATE,
+		//Id:             "1",
+		Name:           "InternalSecurityGroupRulelnfo",
+		Description:    "InternalSecurityGroupRulelnfo_description",
+		Ethertype:      "5",
+		Direction:      "6",
+		Protocol:       "7",
+		PortRange:      "8",
+		RemoteGroupId:  "9",
```

Comment:
@yanmo96 What is this file `network_create_try.go` for? if it's a test file, please rename it as `XXX_test.go`